packages/desktop-client/src/hooks/useSelected.tsx

```diff
@@ -65,7 +64,7 @@ export function useSelected<T extends Item>(
           const selectedItems = new Set(state.selectedItems);
           const { id, event } = action;
 
-          if (event.shiftKey && selectedRange) {
+          if (event && event.shiftKey && selectedRange) {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏why are you changing this? Is it causing any issues? If so: how can I reproduce?

Reply:
As the if !action.event check is removed above, event can now be null, and thus this section now requires a null check. 
(lines:277-279)

Reply:
No longer changed